### PR TITLE
(CLOSED) Upgrade to Django 1.11

### PR DESCRIPTION
**Pull request by kdmccormick**
@tmwbook I looked over the changes and the seem all good to me. I'm going to test it once I get home. If you could look over your changes and let me know when you've tested to your satisfaction that'd be great.

### CODE REVIEW

File: SigmaPi/SigmaPi/settings.py [hunk 1]
```diff
@@ -10,7 +9,7 @@
 ADMINS = environment_settings.ADMINS
 DATABASES = environment_settings.DATABASES
 ALLOWED_HOSTS = environment_settings.ALLOWED_HOSTS
-SENDFILE_BACKEND = environment_settings.SENDFILE_BACKEND
+DOWNLOADVIEW_BACKEND = environment_settings.DOWNLOADVIEW_BACKEND
```
Comment (kdmccormick) [inline SigmaPi/SigmaPi/settings.py]: what's the reasoning for the change here?

Comment (tmwbook) [inline SigmaPi/SigmaPi/settings.py]: For documentation sake, django_sendfile became deprecated as of 1.9, and changed in favor of django-download view

**Comment by kdmccormick** [comment]
Closed in favor of #61

**Comment by kdmccormick** [comment]
Closed in favor of #62

**Pull request closed without merging**